bin/upload-to-cdn.js

```diff
@@ -160,6 +161,16 @@ async function main (packageName, version) {
     ? await getRemoteDistFiles(packageName, version)
     : await getLocalDistFiles(packagePath)
 
+  if (packageName === 'uppy') {
+    // Create downloadable zip archive
+    const zip = new AdmZip()
+    for (const [filename, buffer] of files.entries()) {
+      zip.addFile(filename, buffer, 'entry comment goes here')
```

Comment:
is the comment required? would be preferable to omit it instead of having a dummy value :smile: